frontend/src/component/ui/Toggle.tsx

```diff
@@ -44,19 +44,26 @@ export class Toggle extends Component<Props, State> {
   render() {
     const { enabled } = this.state;
     return (
-      <div className={styles.toggle}>
+      <div
+        className={`${withCSSModule(this.props.styles, styles)} ${
+          styles.toggle
+        }`}
+      >
         <div
-          className={classNames({
+          className={`${withCSSModule(this.props.styles, styles)} ${classNames({
             [styles.background]: true,
             [styles.active]: enabled
-          })}
+          })}`}
           onClick={this.handleClick}
         >
           <div
-            className={classNames({
+            className={`${withCSSModule(
+              this.props.styles,
+              styles
```

Comment:
Is this needed to style the `Toggle`?

Reply:
No. Only background is styled right now. Should this be removed?

Reply:
Yes, please. We will add it back when needed!